jboss/container/wildfly/s2i/2.0/legacy/module.yaml

```diff
@@ -24,6 +24,8 @@ envs:
   description: "DEPRECATED API. A comma separated list of custom galleon feature-packs GAV.  Can be used when a set of Galleon layers has been set in GALLEON_PROVISION_LAYERS."
 - name: GALLEON_PROVISION_LAYERS
   description: "DEPRECATED API. Used during s2i to provision a custom galleon server. Comma separated list of layers to compose the configuration."
+- name: GALLEON_PROVISION_CHANNELS
+  description: "DEPRECATED API. Used during s2i to provision a custom galleon server. Comma separated list of Channel manifest maven coordinates ('GroupId:ArtifactId:[Version]' or yaml file URL (e.g.: file://tmp/manifes.yaml) to resolve artifacts."
```

Comment:
unclosed `(`

\+

is the starting `[` in the correct position? Isn't the last `:` supposed  to be part of the optional version? In tests it is used without `:`.
`'GroupId:ArtifactId:[Version]'`

Reply:
also, this maybe just my perception or bad English but` yaml file URL (e.g.: file://tmp/manifes.yaml)` seems to me a bit ambiguous - it can be read as only `file://` urls are allowed, which is not the case

Reply:
there is still a lonely opening `(` without a closing friend in
`('GroupId:ArtifactId[:Version]' or URL (e.g.: file://tmp/manifes.yaml) to resolve artifacts.`

Reply:
Houps...Fixed. Thanks.